open-craft/opencraft
# [SE-3825] Make ansible appserver values configurable

This PR resolves the issues identified during the investigation of why Watched Forks related PRs are not working properly for master builds by adding ansible configuration values to Watched Forks too.

**Dependencies**: None

**Screenshots**: 

![Screenshot_2021-01-04 Add watched fork OpenCraft](https://user-images.githubusercontent.com/19173947/103532010-c60b1300-4e8a-11eb-8e4d-230b2c337e7e.png)

**Merge deadline**: None, but sooner is better

**Testing instructions**:

1. Checkout changes locally (or on stage)
2. Run migrations
3. Start/Restart OCIM
4. Go to watched forks admin (`/admin/pr_watch/watchedfork/add/`)
5. Create a new Watched Fork to see the model can be saved
6. Create a new Watched Fork entry based on what we have in production for `edx-platform` but for `open-craft/edx-platform`
7. Open a PR for `open-craft/edx-platform`
8. Validate that the instance is created with the expected ansible settings you set on the Watched Fork
9. Revert the changes on Stage (if tested on stage)
10. Unapply migrations (if tested on stage)

**Author notes and concerns**:

N/A

**Reviewers**
